lib/cdo/data/logging/rake_task_event_logger.rb

```diff
@@ -1,13 +1,14 @@
 require lib_dir 'cdo/data/logging/timed_task_with_logging'
+require 'cdo/aws/metrics'
+
 class RakeTaskEventLogger
   STUDY_TABLE = 'rake_performance'.freeze
-  CURRENT_LOGGING_VERSION = 'v0'.freeze
-
+  CURRENT_LOGGING_VERSION = 'v1'.freeze
   def initialize(rake_task)
     @start_time = 0
     @end_time = 0
     @rake_task = rake_task
-    @enabled = !([:development, :test].include?(rack_env))
+    @enabled = !([:development].include?(rack_env))
```

Comment:
If we're enabling this when the rack / rails environment is `test`, we need this logic to work in 3 different types of deployments of our application:

1. `test.code.org` / `test-studio.code.org` - Our "managed" test server. This server has an IAM Profile attached to it that has permissions to PUT records to this Firehose stream
2. Continuous Integration - Drone environments execute with rack / rails environment set to `test` but have very few API keys available. Do they have permissions to PUT records to this Firehose stream÷
3. local development - One an engineer executes tests on their local development environment, they execute with rack / rails environment set to `test. Engineer development environments can assume the employee's AWS developer role and likely have permissions to PUT records to this Firehose stream.

Reply:
Looks like Continuous Integration ("Drone") use case is the only one that lacks permissions needed to PUT to this Firehose stream. The environment variable `CI` can be used to detect that we're executing in a continuous integration (Drone) build and skip certain logic. Here's [an example](https://github.com/code-dot-org/code-dot-org/blob/86752ee25fbfc67678e04658a85b7f5ecc4ce57c/dashboard/app/helpers/levels_helper.rb#L690).

Alternately, we could add permissions to PUT to this Firehose stream to the Role attached to the Drone worker EC2 Instances.